From 8d0d3ec6c7c49fbf99d8baa0e619565236652d4f Mon Sep 17 00:00:00 2001
From: "Bernhard M. Wiedemann" <bwiedemann@suse.de>
Date: Tue, 2 Jul 2024 11:05:21 +0200
Subject: [PATCH] Do not record the build hostname

to make builds reproducible.
See https://reproducible-builds.org/ for why this is good.

This patch was done while working on reproducible builds for openSUSE, sponsored by the NLnet NGI0 fund.
---
 CMakeLists.txt            | 1 -
 include/blas/defines.h.in | 1 -
 2 files changed, 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index f2fbd695..77e4e7e3 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -481,7 +481,6 @@ if (true)
         string( APPEND blaspp_header_defines "#define ${def}\n" )
     endforeach()
 
-    cmake_host_system_information( RESULT HOSTNAME QUERY HOSTNAME )
     # ctime format: Mon Nov 16 15:19:47 2020
     string( TIMESTAMP datetime "%a %b %d %H:%M:%S %Y" )
 
diff --git a/include/blas/defines.h.in b/include/blas/defines.h.in
index f337139f..f5ac9669 100644
--- a/include/blas/defines.h.in
+++ b/include/blas/defines.h.in
@@ -8,7 +8,6 @@
 
 // auto-generated by: @argv@
 // @CMAKE_COMMAND@ @CMAKE_VERSION@
-// host: @HOSTNAME@
 //
 // Definitions for:
 // CXX   = @CXX@
